src/store/options.js

```diff
@@ -75,8 +74,12 @@ export default Options;
 
 chrome.runtime.onMessage.addListener((message) => {
   if (isUpdateOptionsMessage(message)) {
-    store.clear(Options, false);
-    store.get(Options);
+    const options = store.get(Options);
```

Comment:
(Not requesting a change, but only for understanding) What is the effect of the steps (get and if not pending, then clear and get)?